src/ert/config/queue_config.py

```diff
@@ -151,18 +189,68 @@ def validate(self, mem_str_format: str) -> bool:
 
 
 queue_memory_usage_formats: Mapping[str, QueueMemoryStringFormat] = {
-    "SLURM": QueueMemoryStringFormat(suffixes=["", "K", "M", "G", "T"]),
-    "TORQUE": QueueMemoryStringFormat(suffixes=["kb", "mb", "gb", "KB", "MB", "GB"]),
+    QueueSystem.SLURM: QueueMemoryStringFormat(suffixes=["", "K", "M", "G", "T"]),
+    QueueSystem.TORQUE: QueueMemoryStringFormat(
+        suffixes=["kb", "mb", "gb", "KB", "MB", "GB"]
+    ),
+}
+valid_options: Dict[str, List[str]] = {
+    QueueSystem.LOCAL.name: [field.name.upper() for field in fields(LocalOptions)],
+    QueueSystem.LSF.name: [field.name.upper() for field in fields(LsfOptions)],
+    QueueSystem.SLURM.name: [field.name.upper() for field in fields(SlurmOptions)],
+    QueueSystem.TORQUE.name: [field.name.upper() for field in fields(TorqueOptions)],
+    QueueSystem.GENERIC.name: [field.name.upper() for field in fields(QueueOptions)],
 }
 
 
+def _log_duplicated_queue_options(queue_config_list: List[List[str]]) -> None:
+    processed_options: Dict[str, str] = {}
+    for queue_system, option_name, *values in queue_config_list:
+        value = values[0] if values else ""
+        if (
+            option_name in processed_options
+            and processed_options.get(option_name) != value
+        ):
+            logging.info(
+                f"Overwriting QUEUE_OPTION {queue_system} {option_name}:"
+                f" \n Old value: {processed_options[option_name]} \n New value: {value}"
+            )
+        processed_options[option_name] = value
+
+
+def _raise_for_defaulted_invalid_options(queue_config_list: List[List[str]]) -> None:
+    # Invalid options names with no values (i.e. defaulted) are not passed to
+    # the validation system, thus we neeed to catch them expliclitly
+    for queue_system, option_name, *_ in queue_config_list:
+        if option_name not in valid_options[queue_system]:
+            raise ConfigValidationError(
```

Comment:
I guess this won't be an issue when running `LOCAL_QUEUE`; ie. when they won't remove settings for the sake of testing but they will change `QUEUE_SYSTEM` to local.

Reply:
It is to fail hard whenever you have e.g. `QUEUE_OPTION LSF BLABLABLAOPTION foovalue` in your config.